components/script/dom/htmlscriptelement.rs

```diff
@@ -274,7 +274,8 @@ impl HTMLScriptElement {
 
         let window = window_from_node(self);
         let window = window.r();
-        let base_url = window.get_url();
+        let document = document_from_node(self);
+        let base_url = document.base_url();
```

Comment:
Is `document` being used anywhere else afterwards? If not, we can just use a one-liner (i.e. `let base_url = document_from_node(self).base_url();`)